modules/op-release-notes-1-17.adoc

```diff
@@ -292,3 +292,18 @@ With this update, {pipelines-title} General Availability (GA) 1.17.1 is availabl
      value: '--build-arg EXAMPLE="abc def"'
 # ...
 ----
+
+[id="release-notes-1-17-2_{context}"]
+== Release notes for {pipelines-title} General Availability 1.17.2
+
+With this update, {pipelines-title} General Availability (GA) 1.17.2 is available on {OCP} 4.12, 4.14, 4.15, 4.16, 4.17, and 4.18 versions.
```

Comment:
```suggestion
With this update, {pipelines-title} General Availability (GA) 1.17.2 is available from {OCP} 4.12 to {OCP} 4.18.
```

Previous release stated "and later versions" but maybe it should also be more precise?